services/MerchStoreService.ts

```diff
@@ -903,6 +905,15 @@ export default class MerchStoreService {
     };
   }
 
+  private static toEventCalendarInfo(pickupEvent: OrderPickupEventModel): OrderPickupEventCalendarInfo {
+    return {
+      start: pickupEvent.start.getTime(),
```

Comment:
What are your thoughts on adding location as a field to OrderPickupEvents?

Reply:
After discussing, we've decided to add it and copy over the location from the linkedEvent if none is provided. One potential issue is that if the linkedEvent location updates, this field will not, however, we do not think this will be a big problem as the user will be notified through discord.